apps/docs/src/components/LayoutMain.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+
+import clsx from 'clsx'
+import { usePathname } from 'next/navigation'
+
+import { FooterMain } from '@/components/Footer'
+import { Banner } from '@/components/Banner'
+import { useUser } from '@/utils/useUser'
+import { HeaderMain } from './HeaderMain'
+
+export function LayoutMain({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  const { user } = useUser()
+  const pathname = usePathname()
+  const relativePathname = pathname?.replace(new RegExp('^/docs'), '')
+  const shouldShowBanner = user?.pricingTier.isPromo
```

Comment:
What is this logic based on? I think we don't do this anymore, so basically this is always `false`

Reply:
This is straight out copy + pasted from here: https://github.com/e2b-dev/E2B/blob/32859697aa02a6163babe45fd5347a85abe071ac/apps/docs/src/components/Layout.tsx#L24